vchan/libvchan.h

```diff
@@ -47,6 +47,18 @@ libvchan_t *libvchan_server_init(int domain, int port, size_t read_min, size_t w
 
 libvchan_t *libvchan_client_init(int domain, int port);
 
+/* An alternative path for client connection:
+ * 1. Call libvchan_client_init_async first.
+ * 2. Call libvchan_client_init_fd_for_select() and wait for the FD to be readable.
+ * 3. When readable, call libvchan_client_init_async_finish().
+ *
+ * Repeat steps 2-3 until libvchan_client_init_async_finish returns 0. Abort on
+ * negative values (error).
```

Comment:
Do specific negative or positive return values have any meaning?  Right now, a negative return value means _something_ went wrong, but provides no information as to _what_ went wrong.

Reply:
I'd say, "same as libvchan_client_init()". It also isn't explicitly defined, but those error codes.
As for positive value, the only used for now will be 1, meaning "wait more".

Reply:
This mostly proxies errors from libxenvchan, which uses `-1` and may sets errno (but I'm not sure if in all cases). Other than that, a message on stderr may be printed.